axelor-quality/src/main/resources/domains/QualityPictures.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<domain-models xmlns="http://axelor.com/xml/ns/domain-models" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://axelor.com/xml/ns/domain-models http://axelor.com/xml/ns/domain-models/domain-models_5.0.xsd">
+
+	<module name="quality" package="com.axelor.apps.quality.db"/>
+
+	<entity name="QualityPictures">
+
+		<string name="name" title="Name"/>
+		<many-to-one name="content" title="File" ref="com.axelor.meta.db.MetaFile"/>
```

Comment:
name of field must be metaFile not 'content'.

Reply:
Updated